OrleansDashboard.Core/OrleansDashboard.Core.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-	<TargetFramework>net7.0</TargetFramework>
+	<TargetFramework>net8.0</TargetFramework>
     <RootNamespace>OrleansDashboard</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Orleans.Runtime" Version="7.2.0" />
-    <PackageReference Include="Microsoft.Orleans.Sdk" Version="7.2.0" />
+    <PackageReference Include="Microsoft.Orleans.Runtime" Version="8.0.0" />
```

Comment:
Should an `OrleansPackageVersion` be defined in Directory.Build.props to make keeping this version consistent across the different Orleans packages in the various .csproj files?

Reply:
Added https://github.com/OrleansContrib/OrleansDashboard/issues/416 for this work.